udata_hydra/routes/checks.py

```diff
@@ -22,3 +28,26 @@ async def get_all_checks(request: web.Request) -> web.Response:
     if not data:
         raise web.HTTPNotFound()
     return web.json_response([CheckSchema().dump(dict(r)) for r in data])
+
+
+async def create_check(request: web.Request) -> web.Response:
+    """Create a new check"""
+    try:
+        payload: dict = await request.json()
+        url = payload["url"]
```

Comment:
I find it weird to ask for both the resource's URL and id, and I guess it could be error-prone (what happens if you mismatch URL and id?). Wouldn't there be a way to manage with only the id? We could get the URL from the db or data.gouv directly 🤔